library/core/src/fmt/builders.rs

```diff
@@ -186,7 +185,7 @@ impl<'a, 'b: 'a> DebugStruct<'a, 'b> {
     ///     "Bar { bar: 10, .. }",
     /// );
     /// ```
-    #[unstable(feature = "debug_non_exhaustive", issue = "67364")]
+    #[stable(feature = "debug_non_exhaustive", since = "1.51.0")]
```

Comment:
```suggestion
    #[stable(feature = "debug_non_exhaustive", since = "1.52.0")]
```

Note that the current nightly version is 1.52.0.

Reply:
Oh no! I always find it hard to tell exactly where the boundary is!

Reply:
@guswynn Tip: the version currently being developed for can be found in [`src/version`](https://github.com/rust-lang/rust/blob/master/src/version).